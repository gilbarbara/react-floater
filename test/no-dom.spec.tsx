/**
 * @vitest-environment node
 */

import * as React from 'react';
import { renderToStaticMarkup } from 'react-dom/server';

import ReactFloater from '../src';

describe('without dom', () => {
  it('should render', () => {
    const view = renderToStaticMarkup(
      <ReactFloater content="Hello! This is my content!" id="r1tc2">
        Hey
      </ReactFloater>,
    );

    expect(view).toMatchSnapshot();
  });
});
