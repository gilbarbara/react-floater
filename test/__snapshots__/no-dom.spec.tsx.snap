// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`without dom should render 1`] = `
<span data-id="r1tc2"
      style="cursor:help"
>
  Hey
</span>
`;
